runtest.py

```diff
@@ -66,17 +68,31 @@ def __init__(self, **kwargs):
         self.cookies_settings = self.settings_json.get('cookies', {})
         self.extensions_settings = self.settings_json.get('extensions', {})
         self._firefox_profile_path = ''
+        self.suite_upload_dp = ''
         if self.settings_json:
             self._firefox_profile_path = self.firefox_profile_creator.get_firefox_profile(
                 prefs=self.settings_prefs,
                 cookies_settings=self.cookies_settings,
                 extensions_settings=self.extensions_settings)
 
+    def setup(self):
+        upload_dir = os.path.join(os.getcwd(), 'upload')
+        date_seq_id = datetime.strftime(datetime.now(), '%Y%m%d%H%M%S')
+        suite_upload_dir = os.path.join(upload_dir, date_seq_id)
```

Comment:
suggest change to `self.suite_upload_dp = os.path.join(upload_dir, date_seq_id)` and drop line 88